.changelog/4550.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: Added the ability to have a special value for the hidden attribute of a script argument.
```

I feel this better corresponds with the agreed [changlog templates](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs). 
```suggestion
- description: Added support for a special value in the hidden attribute of a script argument.
```